.github/workflows/reusable_fast.yml

```diff
@@ -122,10 +189,9 @@ jobs:
       working-directory: ${{env.BUILD_DIR}}
       run: ctest --output-on-failure --test-dir test -C Release
 
-    # TODO: We could add some script to verify metadata of dll's (selected fields, perhaps)
-    # ref. https://superuser.com/questions/381276/what-are-some-nice-command-line-ways-to-inspect-dll-exe-details
     - name: Print metadata of our dll's
       if: matrix.os == 'windows-latest'
       run: |
         get-command ${{github.workspace}}/build/bin/Release/umf.dll | format-list
         get-command ${{github.workspace}}/build/src/proxy_lib/Release/umf_proxy.dll | format-list
+        
```

Comment:
something wrong here with EOF - there's should be just a single empty line at the end